test/e2e/mock/dl.js

```diff
@@ -0,0 +1,5 @@
+/* eslint-disable camelcase */
+export const score_ad = (adMetadata, bid, auctionConfig, trustedScoringSignals, browserSignals) => {
+	console.log('scoring bid...');
```

Comment:
do we need the console statements in here for a mock?

Reply:
its helpful when loaded in the browser for debugging and doesn't effect the command line. I can comment them out but that seems to provide no benefit and a small hassle when they are needed.